README.md

```diff
@@ -2,79 +2,74 @@
 
 > **General disclaimer:** Please be aware that this template is in public preview. Therefore, expect smaller bugs and issues when working with the solution. Please submit an Issue in GitHub if you come across any issues that you would like us to fix.
```

Comment:
We are in private preview. Once we flick on public for repo then we will be public preview.